docs/syntax.md

```diff
@@ -103,13 +101,13 @@ These options apply to any supported cloud provider.
 
 | <img width=1200 /> | |
 | ---------- | ------------------ |
-| `--nodes` *number_of_nodes*     | Number of nodes to create or add to the cluster. |
+| `--nodes` *number_of_nodes* | Number of nodes to create or add to the cluster. |
 | `--node-cache-size` *size_in_GB* | Size of the data cache space for each node (in GB). Use this to automatically set `--data-disk-count` and `--data-disk-size`, or use those two settings to set each manually. All nodes will have identical storage capacity. Read [Initial Configuration for the New Cluster](using_vfxt_py.md#initial-configuration-for-the-new-cluster) for additional information. |
 | `--data-disk-count` *number_of_disks* | Number of data disk volumes per cluster node. This option can be used with `--create` or with `--add-nodes`. <br/>You can use `--node-cache-size` to set this automatically. Read [Initial Configuration for the New Cluster](using_vfxt_py.md#initial-configuration-for-the-new-cluster) for additional information.  |
 |`--data-disk-size` *size_in_GB* | Size of data disk volumes to create for the vFXT cluster. This option can be used with `--create` or with `--add-nodes`. <br/>You can use `--node-cache-size` to set this automatically. Read [Initial Configuration for the New Cluster](using_vfxt_py.md#initial-configuration-for-the-new-cluster) for additional information.     |
-| `--data-disk-type` *volume_type* | Type of storage volumes to create for the vFXT cluster cache. Azure supports only one disk type (premium LRS) and does not use this option. |
+| `--data-disk-type` *volume_type* | Type of storage volumes to create for the vFXT cluster cache. Values depend on the cloud provider; Azure supports only one disk type (premium LRS) and does not use this option. |
```

Comment:
Looks like I missed this one - the changed text should be removed (older file is correct).